### PR TITLE
Add MUI and restyle/refactor task list + add task component

### PR DESCRIPTION
**Pull request by kathrynherod**
image

### CODE REVIEW

File: client/package.json [hunk 1]
```diff
@@ -9,6 +9,10 @@
     "preview": "vite preview"
   },
   "dependencies": {
+    "@emotion/react": "^11.10.6",
+    "@emotion/styled": "^11.10.6",
+    "@mui/icons-material": "^5.11.11",
+    "@mui/material": "^5.11.12",
```
Comment (kathrynherod) [inline client/package.json:15]: add MUI and dependencies

File: client/src/components/Task.jsx [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  Box,
+  Button,
+  Card,
+  CardActions,
+  CardContent,
+  Typography
+} from '@mui/material'
+
+// Documentation on Material UI Card: https://mui.com/material-ui/react-card/
+function Task ({ task }) {
```
Comment (kathrynherod) [inline client/src/components/Task.jsx:12]: new Task component and all its props

File: client/src/components/TaskList.jsx [hunk 3]
```diff
@@ -1,17 +1,13 @@
...
-    return (
-        <ul style={style}>
-            {taskItems}
-        </ul>
-    )
+import Task from './Task'
+
+function TaskList ({ tasks }) {
+  return (
+    <Grid container spacing={3} xs={12} sx={{m: 3}}>
```
Comment (kathrynherod) [inline client/src/components/TaskList.jsx:7]: slight refactor to use MUI grid and pass props to new Task component

**Comment by jtywa** [comment]
Looks good to me!

**Review by jtywa — APPROVED** [review]

**Pull request merged**